.circleci/config.yml

```diff
@@ -1,21 +1,216 @@
-# Golang CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-go/ for more details
 version: 2.1
+
+aliases:
+  - &filter-only-version-tag
+    tags:
+      only: /^v[0-9]+\.[0-9]+\.[0-9]+.*/
+    branches:
+      ignore: /.*/
+  - &filter-version-not-release
+    tags:
+      only: /^v[0-9]+\.[0-9]+\.[0-9]+.*/
+    branches:
+      ignore: 
+        - /release\/.*/
+        - master
+
+orbs:
+  codecov: codecov/codecov@1.0.5
+
 executors:
-  build_base:
+  test-executor:
+    working_directory: /go/src/github.com/klaytn/klaytn
     docker:
       - image: klaytn/build_base:1.0-go1.13.3-solc0.4.24
-  rpmbuild:
+  rpm-executor:
+    working_directory: /go/src/github.com/klaytn/klaytn
     docker:
-      - image: klaytndevops/circleci-rpmbuild:1.13.3
+      - image: klaytn/circleci-rpmbuild:1.13.3-4
+  default:
+    working_directory: /go/src/github.com/klaytn/klaytn
+    docker:
+      - image: circleci/golang:latest
```

Comment:
Is it okay to use golang:latest always by default?

Reply:
this image will run for tagging, open PR, echo.. etc. do nothing about build/package.
but, when it makes git tag, golang will used to check verison from version.go. 
If checking version by version.go will highly affected by go version, I can specify version number